File: src/com/classe/solaire/Panneau.java
package com.classe.solaire;

import java.lang.Integer;
import java.lang.Object;
import java.lang.String;

public class Panneau {
  private Integer idpanneau;

  private String nompanneau;

  private Object puissance;

  public Panneau(Integer idpanneau, String nompanneau, Object puissance) {
    this.idpanneau = idpanneau;
    this.nompanneau = nompanneau;
    this.puissance = puissance;
  }

  public Integer getIdpanneau() {
    return idpanneau;
  }

  public void setIdpanneau(Integer idpanneau) {
    this.idpanneau = idpanneau;
  }

  public String getNompanneau() {
    return nompanneau;
  }

  public void setNompanneau(String nompanneau) {
    this.nompanneau = nompanneau;
  }

  public Object getPuissance() {
    return puissance;
  }

  public void setPuissance(Object puissance) {
    this.puissance = puissance;
  }
}

File: src/com/cs/model/solaire/Panneau.cs
using System;
namespace src.com.cs.models {
public class Panneau {
 {
    private int idpanneau;
    private string nompanneau;
    private object puissance;

    public Panneau(int idpanneau, string nompanneau, object puissance) {
        this.idpanneau = idpanneau;
        this.nompanneau = nompanneau;
        this.puissance = puissance;
    }
    public int Idpanneau {
        get { return idpanneau; }
        set { idpanneau = value; }
    }
    public string Nompanneau {
        get { return nompanneau; }
        set { nompanneau = value; }
    }
    public object Puissance {
        get { return puissance; }
        set { puissance = value; }
    }
}
    }
}